data/adopters.yaml

```diff
@@ -36,3 +36,274 @@
     url: https://www.pingcap.com/blog/use-case-tidb-in-eleme/
   - language: Chinese
     url: https://www.pingcap.com/cases-cn/user-case-eleme-1/
+- name: Xiaomi
+  url: https://www.mi.com/
```

Comment:
```suggestion
  url: https://en.wikipedia.org/wiki/Xiaomi
```

Reply:
Sorry, the link in adopters.md of the docs repo is wrong. I will also change the link in the docs repo.
We prefer the Wikipedia link here. If the company Wikipedia link doe not exist, we will use its Crunchbase link. If the the company Crunchbase link does not exist, we will use the official website link of this company.